frontend/apps/filemanager/filemanager.lua

```diff
@@ -243,8 +244,18 @@ function FileManager:init()
                     end,
                 }
             },
+            {},
```

Comment:
Yes, it's a little hackish admittedly but in lieu of a proper solution it's a nice way to create the intended visual effect. It's also used in History.

![screenshot_2017-04-10_11-18-09](https://cloud.githubusercontent.com/assets/202757/24854660/8d9019b0-1ddf-11e7-8904-03a0300899fe.png)